src/commcare_cloud/ansible/roles/citusdb/meta/main.yml

```diff
@@ -3,7 +3,7 @@ dependencies:
   - role: monit
   - role: postgresql_base
     postgresql_version: '11'
-    postgresql_cluster_name: '11/main'
+    postgresql_cluster_name: 'main'
```

Comment:
I think this is the reason I was having the issue yesterday @NitigyaS. This was making it test paths like `/opt/data/postgresql/11/11/main/pg_log` instead of `/opt/data/postgresql/11/main/pg_log`